src/components/rubric-levels-mobile.js

```diff
@@ -0,0 +1,1037 @@
+		return scroll > 0;
+	}
+
+	_canScrollRight(scroll, visibleWidth) {
+		const scrollContainer = this.shadowRoot && this.shadowRoot.querySelector('.levels-container');
+		if (!scrollContainer) {
+			return false;
+		}
+		const contentWidth = scrollContainer.scrollWidth;
+		return scroll < contentWidth - visibleWidth - 1;
+	}
+
+	_checkLevelAt(index) {
+		this.focus();
+		this._updateFocused(index);
+		if (!this.readOnly) {
+			this._updateSelected(index);
+		}
+	}
+
+	_checkRadio(radio) {
+		if (this._selected !== -1) {
+			if (this._focused !== -1) {
+				this._animating = true;
+			}
+			if (this.readOnly) {
+				// Rubrics in preview mode still need to block the focus handler from firing for a single frame
+				requestAnimationFrame(() => {
+					this._animating = false;
+				});
+			}
+		}
+		const dataIndex = parseInt(radio.attributes.dataIndex.value);
+		this._updateFocused(dataIndex);
+		if (!this.readOnly) {
+			this._updateSelected(dataIndex);
+		}
+		radio.focus();
+	}
+
+	_getAssessmentHref(assessmentEntity) {
+		return this._getSelfLink(assessmentEntity) || this._getSelfLink(this.entity);
+	}
+
+	_getCriterionCellAriaChecked(index, selected) {
+		return (index === selected).toString();
+	}
+
+	_getCriterionCellHref(criterionCells, index) {
+		if (!criterionCells || !criterionCells[index]) {
+			return '';
+		} else {
+			return this._getSelfLink(criterionCells[index]);
+		}
+	}
+
+	_getCriterionCellTabIndex(index, focused) {
+		return index === (focused === -1 ? 0 : focused) ? 0 : -1;
+	}
+
+	_getCriterionId() {
+		return 'criterion-name';
+	}
+
+	_getIteratorIndex(selected, focused) {
+		return focused !== -1 ? focused : selected;
+	}
+
+	_getLeftIteratorClass(selected, focused, levels) {
+		return classMap({
+			'level-iterator-container': true,
+			disabled: !levels || !this._isLeftIteratorActive(selected, focused, levels)
+		});
+	}
+
+	_getLeftIteratorText() {
+		return getComputedStyle(this).direction === 'rtl'
+			? this.localize('selectNextLevel')
+			: this.localize('selectPreviousLevel');
+	}
+
+	_getLevelClassName(index, selected, hovered, focused, moving) {
+		return classMap({
+			level: true,
+			selected: this._isSelected(index, selected),
+			hovered: this._isHovered(index, hovered),
+			focused: this._isFocused(index, focused) && (this.readOnly || selected === -1),
+			dragging: this._currentDragContext?.elementHasBeenDragged,
+			moving: this._currentDragContext?.elementHasBeenDragged || moving
+		});
+	}
+
+	_getLevelDescription(cell) {
+		const cellEntity = cell && cell.getSubEntityByClass && cell.getSubEntityByClass(Classes.rubrics.description);
+		return cellEntity && cellEntity.properties.text || this.localize('noDescription');
+	}
+
+	_getLevelId(index) {
+		return `level${index}`;
+	}
+
+	_getLevelIndex(index, levelCount) {
+		return getComputedStyle(this).dir === 'rtl'
+			? levelCount - index
+			: index + 1;
+	}
+
+	_getLevelName(level) {
+		return level && level.properties && level.properties.name;
```

Comment:
```suggestion
		return level?.properties?.name;
```
or use `RubricLevelEntity.name`